app/containers/TracksContainer/reducer.js

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * TracksContainer reducer
+ *
+ */
+
+import produce from 'immer';
+import { createActions } from 'reduxsauce';
+
+export const initialState = {
+  trackName: null,
+  tracksData: {},
+  tracksError: null
+};
+
+export const { Types: TracksContainerTypes, Creators: TracksContainerCreators } = createActions({
```

Comment:
naming must be in camelCase here
`trackContainerTypes` and `tracksContainerCreators`